README.md

```diff
@@ -162,8 +162,8 @@ You can tune compression levels as you wish. Default compression levels are `1`.
 Configurable via `local.properties`:
 
 ```
-remote_build.local_gzip_level=1
```

Comment:
It is still a behaviour change. I’m running it locally at this point without any issues, but who knows.